### DISCUSSION TITLE
Is there a way to embed/transclude svg/png automatically?

### DISCUSSION THREAD

**Discussion by aravindk100** [zsviczian/obsidian-excalidraw-plugin#442] [Q&A] [answered]
For e.g. when I use the option, create a new drawing and embed into active pane, is there a way that embed can be made to SVG or PNG as opposed to native format. I am able to change the extension after embed is created, wondering if there is a way to shortcut it. In similar vain is there a way to open the plugin directly from the embedded svg file? 

Embedding svg has been a life saver when it comes to exporting via Typora for example.  Thanks for the amazing plugin.

**Comment by aravindk100** [comment]
Browsing I did find that option which was evading my eyes earlier :) and this appears to solve both SVG by default and editing in Excalidraw. 

Is there any way to make it more native markdown link for embedding as opposed to the wikilink format. My workflow involves for exporting using Typora and wikilink does not work.  I can do this manually but was wondering if this could be an option in the plugin. 

![image](https://user-images.githubusercontent.com/4916255/154342301-2b00a7be-74d7-4e3a-bfad-598947b227e8.png)

**Comment by zsviczian** [comment, marked as answer]
included in Excalidraw 1.6.14
